scripts/release_scripts/update_configs.py

```diff
@@ -154,41 +147,27 @@ def update_app_yaml(
 
 
 def verify_config_files(
-    release_feconf_path: str,
-    release_app_dev_yaml_path: str,
-    verify_email_api_keys: bool
+    release_feconf_path: str, release_app_dev_yaml_path: str
 ) -> None:
     """Verifies that feconf is updated correctly to include
-    mailgun api key, mailchimp api key and redishost.
+    redishos and app.yaml to include correct headers.
```

Comment:
redishost has a typo.

Also should we specify redishost in secrets instead for consistency?

Reply:
Done.

Maybe, later there are much more constants that might be moved to secrets but I want to first analyze the current situation.